### DISCUSSION TITLE
Can I use vercel platform kit on hobby plan?

### DISCUSSION THREAD

**Discussion by thepradipvc** [vercel/community#4241] [Feature Question] [answered]
### Question

Hey,
I have created this project for my portfolio and I need it to be a kind of multi tenant app. I was wondering if I can use vercel hobby plan to create multi tenant app using vercel's platform-kit.

**Comment by swarnava** [comment, marked as answer]
Hi,

Hobby plan is only for testing the platform and not meant to use for Production usecase. Learn more about our Fair Usage Policy here: https://vercel.com/docs/concepts/limits/fair-use-policy

**Comment by thepradipvc** [comment]
Hey,
I understand that the Vercel hobby plan is not for commercial use. I want to deploy my learning project on Vercel as I am learning to deploy a multi-tenant app. So I was wondering if I could do it on hobby plan?

**Reply by swarnava** [reply]
As long as there is no production traffic coming, you should be fine :)

**Reply by thepradipvc** [reply]
However, the Vercel platform kit guide asks to set an environment variable called Vercel team ID which cannot be created in the hobby plan. So I was wondering if it is even possible to create a multi-tenant app on a hobby plan.
